daemon/algod/api/algod.oas2.json

```diff
@@ -523,6 +523,68 @@
         }
       ]
     },
+    "/v2/blocks/{round}/hash": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "schemes": [
+          "http"
+        ],
+        "summary": "Get the block hash for the block on the given round.",
+        "operationId": "GetBlockHash",
+        "parameters": [
+          {
+            "minimum": 0,
+            "type": "integer",
+            "description": "The round from which to fetch block hash information.",
+            "name": "round",
+            "in": "path",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/responses/BlockHashResponse"
+          },
+          "400": {
+            "description": "Bad Request - Non integer number",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Invalid API Token",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "404": {
+            "description": "None existing block ",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "500": {
+            "description": "Internal Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "default": {
+            "description": "Unknown Error"
+          }
+        }
+      },
+      "parameters": [
+        {
+          "type": "string",
```

Comment:
I think this is supposed to be int?

Reply:
also there is a different parameters object defined nested above under "get", I don't know enough about openapi to understand which is for what

Reply:
Yeah it looks like I had a duplicate. I've removed it and the generated code wasn't affected so it seems like it's just been ignoring the one of them.